osf-model/src/main/resources/ontology/osf-model.rdf

```diff
@@ -773,6 +773,65 @@
         <rdfs:range rdf:resource="&xsd;string"/>
     </owl:DatatypeProperty>
 
+    <owl:DatatypeProperty rdf:about="&osfbom;hasContent>
+        <rdfs:label xml:lang="en">hasContent</rdfs:label>
+        <rdfs:comment xml:lang="en">The text of this comment</rdfs:comment>
+        <rdfs:domain rdf:resource="&osfbom;Comment"/>
+        <rdfs:range rdf:resource="&xsd;string"/>
+    </owl:DatatypeProperty>
+
+    <owl:DatatypeProperty rdf:about="&osfbom;isDeleted>
+        <rdfs:label xml:lang="en">isDeleted</rdfs:label>
+        <rdfs:comment xml:lang="en">Whether this comment is deleted</rdfs:comment>
+        <rdfs:domain rdf:resource="&osfbom;Comment"/>
+        <rdfs:range rdf:resource="&xsd;boolean"/>
+    </owl:DatatypeProperty>
+
+    <owl:DatatypeProperty rdf:about="&osfbom;isDeleted>
+        <rdfs:label xml:lang="en">isDeleted</rdfs:label>
+        <rdfs:comment xml:lang="en">Whether this comment is deleted</rdfs:comment>
+        <rdfs:domain rdf:resource="&osfbom;Comment"/>
+        <rdfs:range rdf:resource="&xsd;boolean"/>
+    </owl:DatatypeProperty>
+
+    <!-- "hasNode" is already taken -->
+    <owl:DatatypeProperty rdf:about="&osfbom;associatedNode>
+        <rdfs:label xml:lang="en">associatedNode</rdfs:label>
+        <rdfs:comment xml:lang="en">The node this comment belongs to</rdfs:comment>
+        <rdfs:domain rdf:resource="&osfbom;Comment"/>
+        <rdfs:range rdf:resource="&osfbom;Node"/>
+    </owl:DatatypeProperty>
+
+    <!-- "hasNode" is already taken -->
+    <owl:DatatypeProperty rdf:about="&osfbom;target>
+        <rdfs:label xml:lang="en">target</rdfs:label>
+        <rdfs:comment xml:lang="en">The target of this comment</rdfs:comment>
+        <rdfs:domain rdf:resource="&osfbom;Comment"/>
+        <!-- range may be either comment or node -->
+    </owl:DatatypeProperty>
+
+    <!-- "hasUser" already taken -->
+    <owl:DatatypeProperty rdf:about="&osfbom;madeBy>
```

Comment:
Is `Contributor` an `OSFBusinessObject`?

In my mind, this could go either way. I like having less predicates. But also, I think I am warming up to using `madeBy`. Does a comment "have" a user? maybe madeBy is clearer?